### DISCUSSION TITLE
Why are lines wrapped where they are in `infer_struct001e.fs` and `infer_struct002e.fs`

### DISCUSSION THREAD

**Discussion by jsoref** [dotnet/fsharp#16186] [Q&A] [answered]
These files have not been changed since they were initially published.

https://github.com/dotnet/fsharp/blob/be6352d1b6325ca93edd8487b19b333a81545201/tests/fsharpqa/Source/Conformance/ObjectOrientedTypeDefinitions/TypeKindInference/infer_struct001e.fs#L7-L8

-- fwiw, this first line appears to be the longest `//` line that isn't a `//<` line in this directory

https://github.com/dotnet/fsharp/blob/be6352d1b6325ca93edd8487b19b333a81545201/tests/fsharpqa/Source/Conformance/ObjectOrientedTypeDefinitions/TypeKindInference/infer_struct001e.fs#L10-L11

-- fwiw, the first line here is precisely one character shorter than the first line of the previous block

https://github.com/dotnet/fsharp/blob/be6352d1b6325ca93edd8487b19b333a81545201/tests/fsharpqa/Source/Conformance/ObjectOrientedTypeDefinitions/TypeKindInference/infer_struct002e.fs#L8-L9

-- fwiw, this first line is just as long as the first line of the previous block (which makes it one character shorter than the first block)

* If the lines don't need to be wrapped at all, then I'd propose unwrapping them and adjusting their line numbers
* If the lines do need to be wrapped, but don't need to be wrapped at the precise wrap points (which seem very arbitrary), then I'd propose changing the wrap points to be at word boundaries instead of at the seemingly arbitrary mid-word points.

**Comment by vzarytovskii** [comment]
I suppose if wrapping needs to be there, it depends on terminal width set by tests. Usually, we don't touch tests under `fsharpqa` unless needed, we want to migrate all of them to the `ComponentTests` suite, which has a much better framework for assertions and baseline.

**Reply by jsoref** [reply]
I don't see how the terminal width can change between line 7 and line 10...

That said, if your general policy is to not touch these files, I can ignore them. These things just look kinda odd, and fixing them to be slightly less odd wouldn't be terribly hard (it isn't hard to avoid messing up the line count -- just move letters so that the wrap is before/after a word...)

**Comment by vzarytovskii** [comment, marked as answer]
Wait, the lines you're referring to are likely not used in framework, only ones included in `<Expects>` tag.

**Reply by jsoref** [reply]
So does that mean you'd be amenable to reflowing them?

**Reply by vzarytovskii** [reply]
Sure, or delete them.

**Reply by jsoref** [reply]
opened https://github.com/dotnet/fsharp/pull/16188
